Assets/Scripts/InkDialogSystem.asmdef

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Ink-Dialog-System",
```

Comment:
InkDialogSystem -> InkDialogueSystem here as well for the file name and lib name.